payments/src/components/common/Input/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { forwardRef } from "react";
+import "./index.scss";
+
+interface InputProps {
+  onChange: ({ target }: { target: HTMLInputElement }) => void;
+  type?: string;
+  maxLength?: number;
+  value: string;
+  placeholder?: string;
+  onKeyDown?: (e: React.KeyboardEvent<HTMLInputElement>) => void;
+  testId?: string;
+}
+
+const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => (
+  <input
+    ref={ref}
+    onChange={props.onChange}
+    type={props.type}
+    maxLength={props.maxLength}
+    value={props.value}
+    placeholder={props.placeholder}
+    onKeyDown={props.onKeyDown}
```

Comment:
```suggestion
    {...props}
    onKeyDown={props.onKeyDown}
```
컴포넌트 타입을 작성했을때 spread를 사용하기가 굉장히 편해집니다:)
props를 변경할떄 내부코드를 그대로 두고 interface만 변경함으로써 확장에 용이하죠